src/content/en/tools/chrome-devtools/profile/network-performance/understanding-resource-timing.markdown

```diff
@@ -0,0 +1,222 @@
+---
+layout: shared/narrow
+title: "Understanding Resource Timing"
+description: "It is crucial to understand the phases in which resources are gathered over the network. This is the foundation for fixing load issues."
+published_on: 2016-02-04
+order: 2
+authors:
+  - jonathangarbee
+translation_priority: 0
+key-takeaways:
+  timing:
+    - "Understand the phases of resource timing."
+    - "Know what each phase provides to the Resource Timing API."
+    - "Realize different indicators of performance problems in the timeline graph. Such as series of transparent bars or large green chunks."
+notes:
+  note-tbd:
+    - "TBD note."
+---
+
+<p class="intro">
+  It is crucial to understand the phases in which resources are gathered over the network. This is the foundation for fixing load issues.
+</p>
+
+{% include shared/toc.liquid %}
+
+{% include shared/takeaway.liquid list=page.key-takeaways.emulate-network %}
```

Comment:
Your `TL;DR` is not displaying because this should be `list=page.key-takeaways.timing`